### PR TITLE
Create a python script to triage issues

### PR DESCRIPTION
**Pull request by jlewi**
* An issue is considered triaged if certain criterion such as labels are set.

See: kubeflow/community#280

* Issues needing triage are added to the "Needs Triage" Kanban board.

* Update issue label bot to use "improvement/enhancement" for the enhancement
  type.

Remove partial file.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/code-intelligence/22)

### CODE REVIEW

**Comment by hamelsmu** [comment]
@jlewi can you also add what dependencies you are using in this subfolder so I can run the code?  Either a Docker Container, requirements.txt, or pipenv.  🤞 Hopefully not just a dump of your global environment :)

File: py/code_intelligence/triage.py [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+      if name in ALLOWED_PRIORITY:
+        info.missing_priority = False
+
+        if not name in REQUIRES_PROJECT:
+          info.missing_project = False
+        else:
+          if project_cards:
+            info.missing_project = False
+
+      if name.startswith("area") or name.startswith("communit"):
```
Comment (abhi-g) [inline py/code_intelligence/triage.py]: s/communit/community/ even though it works as is

Comment (jlewi) [reply py/code_intelligence/triage.py]: Done.

**Comment by jlewi** [comment]
@hamelsmu added a requirements.txt file; I might have missed some.

**Pull request merged**